usersession/agent/rest_api.go

```diff
@@ -461,10 +495,17 @@ func postRefreshFinishedNotification(c *Command, r *http.Request) Response {
 		notification.WithUrgency(notification.LowUrgency),
 	}
 
+	if si, err := snap.ReadCurrentInfo(finishRefresh.InstanceName); err == nil {
+		icon = guessAppIcon(si)
+	} else {
+		logger.Noticef("cannot load snap-info for %s: %v", finishRefresh.InstanceName, err)
```

Comment:
minor nitpick, is it normal that a the agent fails to load snap info? if not maybe it makes sense to log this as an internal error?

Reply:
this shouldn't normally happen, but keeping in mind all of this is happening asynchronously snapd may be already removing/reverting/refreshing the snap it notified about earlier, so there's non-0 chance of getting an error here

Reply:
makes sense, thanks!